tests/rust_syscall_tests/src/io.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Traits, helpers, and type definitions for core I/O functionality.
+
+use core::fmt;
+
+struct StdIO<const FD: i32>;
```

Comment:
The casing of this type is really painful. Not sure it's an improvement, but what about just `Stdio`?

Reply:
oops, sorry.. changed..